packages/components/src/components/carousel/carousel-styles.ts

```diff
@@ -178,7 +178,8 @@ export const getComponentCss = (
     'skip-link': {
       opacity: 0,
       pointerEvents: 'none',
-      '&:focus-visible': {
+      // :focus must be used in this case, because :focus-visible is just matched on the focusable element itself, not on the host element.
+      '&:focus': {
         opacity: 1,
         pointerEvents: 'all',
```

Comment:
Couldn't this entire part be defined the other way around without overriding/resetting?
``` 
'skip-link:not(:focus)': {
  opacity: 0,
  pointerEvents: 'none',
} 
```

Reply:
Hey @denyo : 
Do you know why the following code works but the latter not?

```
'skip-link': {
      '&:not(:focus)': {
        opacity: 0,
        pointerEvents: 'none',
      },
    },
```


```
'skip-link:not(:focus)': {
      opacity: 0,
      pointerEvents: 'none',
    },
```